WalletWasabi.Fluent/Views/Wallets/Home/History/Details/CoinJoinDetailsView.axaml

```diff
@@ -27,8 +27,8 @@
       <c:PreviewItem Icon="{StaticResource transaction_id}"
                      Label="Transaction ID">
         <c:PreviewItem DataContext="{Binding TransactionId}" TextValue="{Binding }">
-          <c:PrivacyContentControl NumberOfPrivacyChars="48">
-            <TextBlock Text="{Binding }" Classes="monoSpaced" />
+          <c:PrivacyContentControl NumberOfPrivacyChars="61" MinHeight="18">
```

Comment:
I don't see a difference with or without setting the min height. Other controls still get pushed, and also it doesn't help in this case:
![image](https://user-images.githubusercontent.com/16364053/199969704-70e19873-40ad-4ec9-aeb4-8b3056da9e9f.png)

Shouldn't we just forget it?